winforms/src/toga_winforms/widgets/activityindicator.py

```diff
@@ -0,0 +1,107 @@
+import ctypes
+import io
+from pathlib import Path
+
+import System.Windows.Forms as WinForms
+from PIL import Image, ImageSequence
+from System.Drawing import Image as NetImage
+from System.IO import MemoryStream
+
+from .base import Widget
+
+
+def _get_size(logical_width=32, logical_height=32):
+    user32 = ctypes.windll.user32
+    try:
+        user32.SetProcessDPIAware()
+        dpi = user32.GetDpiForSystem()
+    except AttributeError:  # pragma: no cover
+        dpi = 96
+    physical_width = logical_width * dpi / 96
+    physical_height = logical_height * dpi / 96
+    return int(physical_width * 2.5), int(
+        physical_height * 2.5
+    )  # *2 to make it sharp on monitor
+
+
+def _composite_gif_on_color(path: str, rgb_color: tuple, size: tuple) -> bytes:
```

Comment:
Although it's an entirely internal method, it should have a docstring describing exactly what it does, why, and the arguments/return types.